index.html

```diff
@@ -203,6 +209,66 @@ <h2>Debug Informations</h2>
         <h1> Move Timer: </h1>
         <progress value="0" max="2.5" id="timer-bar"></progress>
     </div>
+    <script>{
+        {
+            document.addEventListener('DOMContentLoaded', () => {
+        const tabButtons = document.querySelectorAll('.tabs button');
+        const highScores = document.getElementById('high-scores');
+        const achievements = document.getElementById('achievements');
+        const instructions = document.querySelector('.sidebar > .selected');
+
+        tabButtons.forEach(button => {
+            button.addEventListener('click', () => {
+                const target = button.getAttribute('data-target');
+
+                // Hide both sections initially
+                highScores.classList.add('hidden');
+                achievements.classList.add('hidden');
+
+                // Show the selected section
+                if (target === 'high-scores') {
+                    highScores.classList.remove('hidden');
+                } else if (target === 'achievements') {
+                    achievements.classList.remove('hidden');
+                }
+            });
+        });
+
+        // Hide both sections when in Instructions
+        instructions.addEventListener('click', () => {
+            highScores.classList.add('hidden');
+            achievements.classList.add('hidden');
+        });
+    });
+
+
+        }
+        document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Why are you adding to the event again, this is already being done above, i believe these can be combined them

Reply:
Lets extract these as separate function and then we can call by adding a listener, also any reason why we have this code in HTML file ?